book/B-embedding-git/sections/go-git.asc

```diff
@@ -0,0 +1,84 @@
+=== go-git
+
+(((go-git)))((("Go")))
+In case you want to integrate Git into a service written in Golang, there also is a pure Go library implmentation. In contrast to using a Libgit2 bindings, this implementation does not have any native dependencies, not prone to memory management errors and is transparent for standard Golang performance analysis tooling like CPU, Memory profilers, race detector, etc.
```

Comment:
Plus there's a typo "implmentation"

Reply:
Thank you for your kind suggestions!

> Let's try to avoid disparaging other libraries.

This of course was not an intention, thank you for noticing and pointing it out.